problems/2023-basic-turbulence-cases/backward-facing-step/bfs_flow_step.i

```diff
@@ -0,0 +1,415 @@
+Re = 3.6e4
+density = 1      # kg cm-3
+D = 1
+inlet = 1
+viscosity = '${fparse density * inlet * D / Re}'    # dynamic viscosity, mu = nu * rho, kg cm-1 s-1
+alpha = .33333           # INS SUPG and PSPG stabilization parameter
+
+[GlobalParams]
+  integrate_p_by_parts = false
+  viscous_form = 'traction'
+  pressure = p
+  velocity = vel
+[]
+
+[Mesh]
+  [block]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '4  .8 .4 1.6 .4 6.8 4  2  2  2.5 3  3.5 25'
+    ix = '64 16 10 50  10 136 64 25 20 20  20 20  125'
+    dy = '.025 .025 .9 .025 .025 .025 .025 .2 1.25 1  2.5 1.5 1.25 .2 .025 .025'
+    iy = '16   8    36 5    10   20   5    10 25   20 5   30  25   10 5    20'
+  []
```

Comment:
I wrote them this way so that the numbers coincide vertically with the numbers in the previous line. `dy` determines the mesh size and `iy` determines the number of mesh elements of each corresponding mesh size entry in `dy`.